fmriprep/workflows/bold.py

```diff
@@ -281,11 +288,13 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
 
     bold_reference_wf = init_bold_reference_wf(omp_nthreads=omp_nthreads)
 
+    # STC on the BOLD
```

Comment:
Please add a comment that the next line **needs** to be like that (`if run_stc is True`). Some linting tools will mark this as a potential error and it is not.

Or be explicit:

```
if run_stc and run_stc != 'TooShort'
```